descriptions/Engine.BKEngine.md

```diff
@@ -0,0 +1 @@
+[**BK Engine**](https://bke.bakery.moe/index_en.html) is a Cross Platform Visual Novel Game Engine. It is free, [**open source**](https://github.com/BKEngine/Creator), and supports Most Desktop and Mobile OS
```

Comment:
```suggestion
[**BK Engine**](https://bke.bakery.moe/index_en.html) is a Visual Novel Game Engine.
```

Reply:
That barely tells anything. "It's an engine"? Yeah, so is everything else in the "Engine" category.

Reply:
No it tells that it's a visual novel engine, which is enough, if someone wants to know more he'll just click on the link